CONTRIBUTORS.yaml

```
@@ -1889,6 +1889,21 @@ pcm32:
     name: Pablo Moreno
     joined: 2019-06
 
+pkulzerUNC:
```

```suggestion
paige-kulzer:
```

the value here and in the FAQ must match